elementor/elementor
# Internal: Update GitHub Community Documentation and Forms [ED-12633]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* **Updated** the Repository's **Readme** file `README.md` with revised text and replaced links with short links
* **Updated** the organization's **Contribution Guidelines** file `.github/CONTRIBUTING.md` with revised text and replaced links with short links
* **Created** the organization's **Code of Conduct** `.github/CODE_OF_CONDUCT.md` file
* **Deleted** the **deprecated** Bug Report issue template file `.github/ISSUE_TEMPLATE.md ` and the deprecated **Feature Request** issue template file `.github/feature-request.md`
* **Updated** the "New Issue's **Choose**" file `.github/ISSUE_TEMPLATE/config.yml` with revised text and replaced links with short links
* **Updated** the **Bug Report Form** template file `.github/ISSUE_TEMPLATE/bug.yml` with revised text and fields for better clarity and to better enforce our requirements for bug reporting
* **Updated** the **Feature Request Form** template file `.github/DISCUSSION_TEMPLATE/feature-request.yml` with revised text and fields for better clarity and to better enforce our requirements for bug reporting
* **Created** a new **Developers Community** template file `.github/DISCUSSION_TEMPLATE/developers-community.yml` to give context and better inform contributors.

## Description
An explanation of what is done in this PR

This is an update to all community-related files as well as the main repository's **readme** file to reflect better our current necessities as well as to replace outdated text and invalid links and add relevant information that was so far missing from those files.

All forms were also revised to reflect our requirements better.


## Test instructions
This PR can be tested by following these steps:

* Visit all documents
* Check if they are appearing correctly
* Review the text
* Test the links

## Quality assurance


Fixes #
